.github/scripts/trymerge.py

```diff
@@ -1256,6 +1256,10 @@ def find_matching_merge_rule(
         raise RuntimeError(reject_reason)
     checks = pr.get_checkrun_conclusions()
     base_rev = None
+    ignore_current_checks = None
+    if skip_current_checks:
+        _, failing = categorize_checks(checks, list(checks.keys()))
+        ignore_current_checks = [x[0] for x in failing]
```

Comment:
Oh man, this raw tuple hurts readability so much 💀

I know it's not your fault, but mind converting it to a NamedTuple?